doc/radosgw/adminops.rst

```diff
@@ -1976,9 +1976,14 @@ Binding libraries
  - `QuentinPerez/go-radosgw`_
 
```

Comment:
Should we add `https://github.com/ceph/go-ceph` while we're here?

Reply:
@anthonyeleven I guess this is addressed in #43085 ?

Reply:
Aye, I had not seen that PR yet when I commented.